docs/dev-tools/zombienet/tutorials/minimal-example.md

```diff
@@ -0,0 +1,229 @@
+            <td>native</td>
+        </tr>
+            <tr>
+                <th colspan="3" align="center">
+                Node Information
+                </th>
+            </tr>
+        <tr>
+            <th>Name</th>
+            <td>alice</td>
+        </tr>
+        <tr>
+            <th>Direct Link</th>
+            <td><a href="https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55308#explorer">https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55308#explorer</a></td>
+        </tr>
+        <tr>
+            <th>Prometheus Link</th>
+            <td><a href="http://127.0.0.1:55310/metrics">http://127.0.0.1:55310/metrics</a></td>
+        </tr>
+        <tr>
+            <th>Log Cmd</th>
+            <td>tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2</td>
+        </tr>
+            <tr>
+                <th colspan="3" align="center">
+                Node Information
+                </th>
+            </tr>
+        <tr>
+            <th>Name</th>
+            <td>bob</td>
+        </tr>
+        <tr>
+            <th>Direct Link</th>
+            <td><a href="https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55312#explorer">https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55312#explorer</a></td>
+        </tr>
+        <tr>
+            <th>Prometheus Link</th>
+            <td><a href="http://127.0.0.1:55314/metrics">http://127.0.0.1:55314/metrics</a></td>
+        </tr>
+        <tr>
+            <th>Log Cmd</th>
+            <td>tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2</td>
+        </tr>
+            <tr>
+                <th colspan="3" align="center">
+                Node Information
+                </th>
+            </tr>
+        <tr>
+            <th>Name</th>
+            <td>collator01</td>
+        </tr>
+        <tr>
+            <th>Direct Link</th>
+            <td><a href="https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55316#explorer">https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55316#explorer</a></td>
+        </tr>
+        <tr>
+            <th>Prometheus Link</th>
+            <td><a href="http://127.0.0.1:55318/metrics">http://127.0.0.1:55318/metrics</a></td>
+        </tr>
+        <tr>
+            <th>Log Cmd</th>
+            <td>tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2</td>
+        </tr>
+        <tr>
+            <th>Parachain ID</th>
+            <td>100</td>
+        </tr>
+        <tr>
+            <th>ChainSpec Path</th>
+            <td>/var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2</td>
+        </tr>
+    </table>
+</div>
+
+!!! note 
+    If the IPs and ports are not explicitly defined in the configuration file, they may change each time the network is started, causing the links provided in the output to differ from the example.
```

Comment:
IPs can also change? Or only the ports?

Reply:
Yes, you can use the keyword `local_ip` in the settings section of the config file. The default is `127.0.0.1`, but it can be changed to any other IP used to expose local services (RPC/metrics/monitors).